manifests/params.pp

```diff
@@ -6,6 +6,7 @@
   $manage_repos       = true
   $download_path      = '/tmp'
   $destination        = '/opt'
+  $bundle_url         = undef
```

Comment:
How is params.pp used?

Reply:
Ok, I paid attention now to your message where you said "I’m not clear on the difference between params.pp".
@reist Do you know?